src/components/KymaCompanion/components/Chat/messages/marked.scss

```diff
@@ -0,0 +1,193 @@
+.markdown {
+  /* Theming */
+  .light {
+    --panel-background: #ebebeb;
+    --panel-text: #333;
+    --formatted-text: #000;
+    --link: #004bfa;
+    --border: #777;
+    --code: rgba(255, 255, 255, 0.8);
+    --table: var(--code);
+    --table-header: rgba(128, 128, 128, 0.2);
+  }
+
+  .dark {
+    --panel-background: #252525;
+    --panel-text: #ccc;
+    --formatted-text: #fff;
+    --link: #7b9ff2;
+    --border: #777;
+    --code: rgba(0, 0, 0, 0.3);
```

Comment:
In dark mode, the `code` is black because of transparency. I asked the guide authors about this color.